.github/workflows/_ci.yaml

```diff
@@ -91,6 +90,16 @@ jobs:
       DOCKERFILE: .github/container/Dockerfile.maxtext.amd64
```

Comment:
I think CONTAINER_NAME should be upstream-maxtext to match the other rosetta frameworks